lib/js/components/PopOver/PopOver.vue

```diff
@@ -128,6 +134,15 @@
 		padding: $space-s;
 		// Override popperjs styles
 		text-align: left;
+		max-height: min(30vh, 400px);
+		overflow: scroll;
+		-ms-overflow-style: none; /* IE and Edge */
+		scrollbar-width: none; /* Firefox */
+
+		/* Hide scrollbar for Chrome, Safari and Opera */
+		&::-webkit-scrollbar {
```

Comment:
hide scrollbar? How users will know that there is more content?

Reply:
The problem with scrollbar is that it's inflicting the styles of Tooltip itself, see - https://miro.com/app/board/uXjVPgNZ25Q=/?moveToWidget=3458764548142477100&cot=14.
We agreed that scrollbars should be hidden at the moment - if Tooltip will contain more information than showed, the user would scroll intuitively.
Nonetheless I will work on making scrollbar visible in next iteration.

Reply:
did you verify it won't affect existing popOvers? Did UX agree on this?

Reply:
Outdated